spec/api-native-image-spec.js

```diff
@@ -105,6 +105,14 @@ describe('nativeImage module', () => {
     return matchingImage
   }
 
+  describe('templateImage property', () => {
+    it('returns whether the image is a template image', () => {
```

Comment:
Can we add tests for the behavior here.  i.e. if you do `nativeImage.createFromPath('MyImage_Template.png`)` then it should be `true`.

If you do `nativeImage.createFromPath('MyImage.png')` it should be `false`.

Only on macOs though

Reply:
This tests the setter, but not the getter returns the correct value on the first try.  

```js
nativeImage.createFromPath('MyImage_Template.png') === true
```

Is a test we want here as well